### PR TITLE
Add hook info in token reference

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds explanation that hooks are supported for token sub. Need to add hooks in example.

## Motivation and Context
Partially addresses https://github.com/sensu/sensu-docs/issues/1884

## Review Instructions
I added content that explains support for hooks in token sub, but I am not sure how to extend the example to include hooks as Jef suggests.

### CODE REVIEW

File: content/sensu-go/5.14/reference/tokens.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ You can use tokens to fine-tune check attributes (like alert thresholds) on a pe
 
 When a check is scheduled to be executed by an agent, it first goes through a token substitution step. The agent replaces any tokens with matching attributes from the entity definition, and then the check is executed. Invalid templates or unmatched tokens will return an error, which is logged and sent to the Sensu backend message transport. Checks with token matching errors will not be executed.
 
-Token substitution is only supported for the [`command` attribute][7] of a check definition, and only [entity attributes][4] are available for substitution. Available attributes will always have [string values](#token-data-type-limitations), such as labels and annotations.
+Token substitution is supported for check definition [`command`][7] attributes and [hooks][8].
```
Comment (cwjohnston) [inline content/sensu-go/5.14/reference/tokens.md]:
```suggestion
Token substitution is supported for check definition [`command`][7] attributes and [hook][8] `command` attributes.
```

**Review by cwjohnston — APPROVED** [review]
One minor suggestion, you can take it or leave it

**Comment by hillaryfraley** [comment]
@cwjohnston  thank you. I'm not sure how to handle this part of Jef's issue (https://github.com/sensu/sensu-docs/issues/1884):
"We should be able to extend the existing check example to include a hook, both making use of entity scoped labels for subsitution"

**Comment by hillaryfraley** [comment]
@jspaleta do you have an example that includes the hook that would make sense here?

**Comment by hillaryfraley** [comment]
Jef is planning to add a suggestion for including hooks in the example in a separate PR, so I'm going to propagate and merge this one so he can submit his PR against the latest and greatest.

**Pull request merged**